sdk/src/main/java/com/kin/ecosystem/KinContext.kt

```diff
@@ -0,0 +1,6 @@
+package com.kin.ecosystem
+
+import android.content.Context
+import android.content.ContextWrapper
+
+class KinContext(context: Context): ContextWrapper(context.applicationContext)
```

Comment:
why is that? to avoid warning?

Reply:
avoid the warning and having the context in `Kin.java` level.